internal/quaiapi/api.go

```diff
@@ -358,7 +358,7 @@ func (s *PublicBlockChainAPI) GetProof(ctx context.Context, address common.Addre
 func (s *PublicBlockChainAPI) GetHeaderByNumber(ctx context.Context, number rpc.BlockNumber) (map[string]interface{}, error) {
 	header, err := s.b.HeaderByNumber(ctx, number)
 	if header != nil && err == nil {
-		response := RPCMarshalETHHeader(header)
+		response := RPCMarshalETHHeader(header.Header()) //TODO: mmtx this function will break once extra fields are stripped from header.
```

Comment:
is this TODO still relevant? What do we do about it?

Reply:
Not sure if this will break anything, will remove the this comment